python/feature_tokenization.py

```diff
@@ -46,17 +47,17 @@
 n_patients = trimmed_seq["medrec_key"].nunique()
 
 # Ensure we're sorted
-trimmed_seq.sort_values(["medrec_key", "pat_key", "dfi"], inplace=True)
```

Comment:
@scotthlee This was kind of an oopsie on my part. I assumed somewhat naively that this was the correct sorting, but actually, it shuffled the days from index. I expected it to have a bigger impact on the LSTM than it actually did, though. We should still probably re-run